thread/common/thrprof.c

```diff
@@ -1025,3 +1030,35 @@ omrthread_get_jvm_cpu_usage_info_error_recovery(void)
 		GLOBAL_UNLOCK_SIMPLE(lib);
 	}
 }
+
+intptr_t
+omrthread_get_self_thread_time(omrthread_thread_time_t *threadTime)
+{
+ #if defined(LINUX)
+	struct rusage rUsage = {0};
+	int rc = getrusage(RUSAGE_THREAD, &rUsage);
+
+	if (-1 == rc) {
+		return -1;
+	}
+
+	threadTime->userTime = (SEC_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_utime.tv_sec)
+							+ (MICRO_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_utime.tv_usec);
+	threadTime->sysTime = (SEC_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_stime.tv_sec)
+							+ (MICRO_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_stime.tv_usec);
```

Comment:
I think lines 1046 and 1048 are indented excessively; suggest one or two tabs more than the preceding line is sufficient in each case.